pkg/daemon/ceph/osd/volume.go

```diff
@@ -49,6 +50,11 @@ const (
 	dbDeviceFlag         = "--db-devices"
 	cephVolumeCmd        = "ceph-volume"
 	cephVolumeMinDBSize  = 1024 // 1GB
+	restoreConPath       = "/usr/sbin/restorecon"
+	restoreConNewContent = `#!/usr/bin/env bash
+echo "original restorecon command is now at %s"
```

Comment:
Does this show up in the osd prepare log? How about a message that indicates restorecon was skipped?

Reply:
No, it only shows up in the `ceph-volume.log` file, let me clarify the message.